### ISSUE TITLE
Cannot access (read/write) the personal private folder (macOS High Sierra)

### ISSUE THREAD

**Issue by ianeq** [keybase/client#8873]
Hello,

I cannot access the private personal folder nor create a git repository on any of the Macs I use and have added and signed with Keybase.

On the CLI level (terminal) I can cd to the /keybase/private/**username** but no list, read or write works. I can easily read/write to one other shared private folder.

"This device has not yet have read access to directory /keybase/private/**username** ... "

Many thanks,
Ianeq

**Comment by strib** [comment]
Hi @ianeq, if you do a `keybase log send` and report the resulting ID here, I can take a look.  If you've revoked all of your original devices, you may have left yourself in a position where no new devices can read it.

**Comment by ianeq** [comment]
The log ID: b68a479a71f6c176fc6f3b1c

**Comment by strib** [comment]
@ianeq: yeah looks like your private folder in only keyed for the devices `IANEQ5` and `PRO1`, both of which have been revoked.  So that means KBFS and git won't work, and unfortunately that means any data in that folder is unrecoverable.  We can reset the folder if you like, to make things work again.

If you want us to reset your folder, please run this command on an currently-valid keybase device, and substitute the current date and time where indicated:

```
keybase sign -m "<DATE_AND_TIME>: Please reset folder /keybase/private/ianeq because all the devices that have access have been revoked."
```

and post the results here.

**Comment by ianeq** [comment]
BEGIN KEYBASE SALTPACK SIGNED MESSAGE. kXR7VktZdyH7rvq v5weRa0zkLOSoc9 eEfacwapzAXIFFb grhiJhmGvqHqi7h fTlgAO9kbWVTHMm oqPAlF4hv0BhOA8 FnlfhTH9lgBNxHw ozoMhxFzTDmrfso r8EZ4I36RhAql6F LDHKYM5aj3d3g9z O1cvrWIdpQ1u1Dq yRE09yihMaCWx1O oZQUyg9HyYM7sYB USKUPmwN7HH2mFg e5tRdN6jMxJ5ld1 ReUj9p6Z85G7GYz TQMbrZa9nWcYEhT AqiNzdQUJWyvqm6 u5ywzkixSX9ZDou hFIvNZPHv8n3B6e WRbREgiXJcu0plM 8zLW83dcBJD9edf WqnileyudUePPv0 Rsbu. END KEYBASE SALTPACK SIGNED MESSAGE.

**Comment by strib** [comment]
I reset the folder.  Please check it out and close this if everything seems ok.  (Might require a Keybase restart.)

**Comment by ianeq** [comment]
Works great, thanks a lot!
